lib/components/dashboard/index.tsx

```diff
@@ -162,6 +167,15 @@ const Dashboard: React.FC<Props> = ({ mode }) => {
     let unmounted = false;
     let source = axios.CancelToken.source();
     if (currPlan && currPlan._id !== 'noPlan') {
+      axios
+        .get(getAPI(window) + '/coursesByPlan/' + currPlan._id)
+        .then((response) => {
+          const sisCourses: SISRetrievedCourse[] = response.data.data;
+          dispatch(updateCourseCache(sisCourses));
+        })
+        .catch((err) => {
+          console.log(err);
+        });
```

Comment:
cache SIS courses associated with plan in one API call 

moved from pages/index.tsx to dashboard/index.tsx